src/main/plugin/iso19139.ca.HNAP/loc/fre/labels.xml

```diff
@@ -2425,6 +2425,10 @@
       <option value="ESRI REST: Map Service">ESRI REST: Map Service</option>
       <option value="OGC:WMS">OGC:WMS</option>
       <option value="OGC:WFS">OGC:WFS</option>
+      <option value="OGC API – Coverages">OGC API – Coverages</option>
```

Comment:
Is there a reason AMQPS is the only one that has French translation?

Reply:
@ianwallen I've added the French translations, but please check to validate them.